examples/electron/electron.js

```diff
@@ -19,6 +19,10 @@ function createWindow() {
         });
     }, [{ label: "Exit", click: (menuItem) => app.quit() }]);
 
+	container.ipc.subscribe("stock.selected", function (event, message) {
+		console.log("Message received: " + message.symbol);
+    });
+
```

Comment:
In order to see this when running the example, be sure to launch Electron with --enable-logging